wyvern/experimentation/providers/eppo_provider.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from typing import List, Optional
+
+import eppo_client  # type: ignore
+from eppo_client.assignment_logger import AssignmentLogger  # type: ignore
+from eppo_client.config import Config  # type: ignore
+
+from wyvern import request_context
+from wyvern.components.events.events import LoggedEvent
+from wyvern.config import settings
+from wyvern.event_logging import event_logger
+from wyvern.exceptions import ExperimentationClientInitializationError
+from wyvern.experimentation.experimentation_logging import (
+    ExperimentationEvent,
+    ExperimentationEventData,
+)
+from wyvern.experimentation.providers.base import (
+    BaseExperimentationProvider,
+    ExperimentationProvider,
+)
+
+
+class EppoExperimentationClient(BaseExperimentationProvider):
+    """
+    An experimentation client specifically for the Eppo platform.
+
+    Extends the BaseExperimentationProvider to provide functionality using the Eppo client.
+
+    Methods:
+    - __init__() -> None
+    - get_result(experiment_id: str, entity_id: str, **kwargs) -> str
+    - log_result(experiment_id: str, entity_id: str, variant: str) -> None
+    """
+
+    def __init__(self):
```

Comment:
nit:
we've been using this pattern within integration code: pass api_key/token as an optional param in the constructor. This will make it easier if ppl just wanna test without some real environment var set up.
```
def __init__(self, api_key: Optional[str] = None):
    api_key = api_key or settings.EPPO_API_KEY
```

Reply:
SG